charts/README.rst

```diff
@@ -10,9 +10,33 @@ Requires minikube, kubectl and helm.
 
     $ minikube start
     $ helm init
+    # Run only once!
+    $ echo "$(minikube ip) renga-k8s.build keycloak.renga-k8s.build gitlab.renga-k8s.build jupyterhub.renga-k8s.build" | sudo tee -a /etc/hosts
```

Comment:
scope this when #172 is solved

Reply:
it's not doing much apart from setting a host for the IP -- AutoDevOps doesn't work without it.